src/pages/signin/SignInPageLayout/SignInPageLayoutNarrow.js

```diff
@@ -17,9 +17,10 @@ const SignInPageLayoutNarrow = ({children}) => (
         <View>
             <View style={[styles.signInPageInnerNative]}>
                 <View style={[styles.signInPageLogoNative]}>
-                    <ExpensifyCashLogoIcon
+                    <Expensicon
+                        name={ICON_NAMES.EXPENSIFY_CASH_LOGO}
                         width={variables.componentSizeLarge}
-                        height={variables.componentSizeNormal}
+                        height={variables.componentSizeLarge}
```

Comment:
I think this was a mistake. Again, the logo isn't really a component, it's more so a branding asset. We also have no other things that are at height 50 that we would consider to be "these are the large size components we use across the app"

So I would remove `componentSizeLarge` and just define the width/height of this as a style, no need for a variable IMO.

Reply:
As discussed 1:1 the `componentSizeLarge` should be handled in a separate PR.